velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -14,12 +14,132 @@
  * limitations under the License.
  */
 
-#include "velox/common/base/tests/GTestUtils.h"
-#include "velox/dwio/parquet/tests/ParquetTpchTestBase.h"
-
-class ParquetTpchTest : public ParquetTpchTestBase {
- public:
-  ParquetTpchTest() : ParquetTpchTestBase() {}
+#include <folly/init/Init.h>
+#include <vector>
+
+#include "velox/common/base/Fs.h"
+#include "velox/common/file/FileSystems.h"
+#include "velox/connectors/tpch/TpchConnector.h"
+#include "velox/dwio/parquet/RegisterParquetReader.h"
+#include "velox/exec/tests/utils/AssertQueryBuilder.h"
+#include "velox/exec/tests/utils/HiveConnectorTestBase.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+#include "velox/exec/tests/utils/TempDirectoryPath.h"
+#include "velox/exec/tests/utils/TpchQueryBuilder.h"
+#include "velox/functions/prestosql/aggregates/RegisterAggregateFunctions.h"
+#include "velox/functions/prestosql/registration/RegistrationFunctions.h"
+#include "velox/parse/TypeResolver.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::exec;
+using namespace facebook::velox::exec::test;
+
+class ParquetTpchTest : public testing::Test {
+ protected:
+  void SetUp() override {
+    duckDb_ = std::make_shared<DuckDbQueryRunner>();
+    tempDirectory_ = TempDirectoryPath::create();
+
+    functions::prestosql::registerAllScalarFunctions();
+    aggregate::prestosql::registerAllAggregateFunctions();
+
+    parse::registerTypeResolver();
+    filesystems::registerLocalFileSystem();
+    parquet::registerParquetReaderFactory();
```

Comment:
Don't we need to register writer factory? CC: @majetideepak 

I'm getting errors:

```
Error Code: INVALID_STATE
Reason: WriterFactory is not registered for format parquet
Retriable: False
Expression: it != writerFactories().end()
Function: getWriterFactory
File: fbcode/velox/dwio/common/WriterFactory.cpp
Line: 55
```

Reply:
@mbasmanova You are right, we should register the Parquet Writer Factory. How was this fixed?

Reply:
I see that you fixed it internally. https://github.com/facebookincubator/velox/commit/9f9fb193e97060bb74eb71df0fb53f33995d7278
Thank you! Looks neat to leave a new line.